libs/content-watcher-lib/src/pubsub/pubsub.config.ts

```diff
@@ -0,0 +1,23 @@
+import { JoiUtils } from '#config';
+import { registerAs } from '@nestjs/config';
+import Joi from 'joi';
+
+export interface IPubSubConfig {
+  webhookMaxRetries: number;
+  webhookRetryIntervalSeconds: number;
+}
+
+export default registerAs('pubsub', (): IPubSubConfig => {
+  const configs: JoiUtils.JoiConfig<IPubSubConfig> = {
+    webhookMaxRetries: {
+      value: process.env.WEBHOOK_FAILURE_THRESHOLD,
```

Comment:
I'm guessing `WEBHOOK_FAILURE_THRESHOLD` and `webhookMaxRetries` are kinda the same? Having **threshold** in here feels confusing if they are the same.

Reply:
Yes. Some of the webhook/health check-related configs are used differently across the different Gateway services; that can be the subject of a whole separate PR.